amp_conf/htdocs/admin/libraries/BMO/Ajax.class.php

```diff
@@ -63,14 +63,15 @@ public function doRequest($module = null, $command = null) {
 			$this->ajaxError(403, 'ajaxRequest declined');
 		}
 
+		$url = parse_url($_SERVER['HTTP_REFERER']);
```

Comment:
Quite happy to use that, but can we guarantee it's form? will it be 127.0.0.1, localhost, raspbx.local ....? Sorry, lack of knowledge on my part here.